### ISSUE TITLE
Installation error message libdav1d.dll not found

### ISSUE THREAD

**Issue by anna452452** [UltraStar-Deluxe/USDX#792]
I am trying to download Ultrastar Deluxe app on my Dell with Windows 10 but getting an error message : 
Code cannot continue because libdav1d.ddl is not found. 
I have tried uninstalling and reinstalling several times without success. 
I apologise if this is a basic problem that can easily be solved - I am in no way computer savvy.

**Comment by s09bQ5** [comment]
Do you maybe have a virus scanner that put the file into quarantine?
No, there is no virus in that file.
Yes, we [already tried to persuade virus scanner vendors](https://github.com/UltraStar-Deluxe/USDX/issues/368#issuecomment-1708306078) that this file is safe and were successful in many cases, but some vendors provide no way to contact them or listen only to their paying customers.

**Comment by basisbit** [comment]
might be related to this recent change: https://github.com/UltraStar-Deluxe/USDX/pull/772

@anna452452 what version of UltraStar Deluxe do you use?

**Comment by s09bQ5** [comment]
> might be related to this recent change: #772

No, that's the uninstaller. It must be listed there. The other DLLs are listed there as well.

**Comment by anna452452** [comment]
I think it must be my computer and some antivirus program because I have just downloaded the program without problems on my old (10 years old!) laptop without a problem. It was the latest version available that I downloaded. 
I have absolutely no idea about computers so I have no idea how to get the antivirus programs to accept files. I will just play it on my old laptop. 
Thank you all for your replies!
